src/core/graph.go

```diff
@@ -19,6 +22,9 @@ type BuildGraph struct {
 	packages *cmap.Map[packageKey, *Package]
 	// Registered subrepos, as a map of their name to their root.
 	subrepos *cmap.Map[string, *Subrepo]
+	// Subincludes that are subincluded by other subincludes
+	subincludeSubincludes map[BuildLabel]labelSet
+	subincMux             sync.RWMutex
```

Comment:
I doubt you get any benefit from RWMutex. I don't think it ever reads in parallel so a normal mutex is probably slightly faster + simpler

Reply:
Oh is it faster? I guess the rw mutex is actually two mutexes?  Yeah, I can change this to a normal mutex.